task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -31,6 +31,7 @@ spec:
           cpu: 10m
       script: |
         #!/usr/bin/env bash
+        set -e
```

Comment:
We cannot fail script, checks must always return `HACBS_TEST_OUTPUT`

Any error handling must happen within the script

Reply:
Remove.